blueflood-core/src/test/java/com/rackspacecloud/blueflood/types/BaseRollupTest.java

```diff
@@ -0,0 +1,17 @@
+package com.rackspacecloud.blueflood.types;
+
+/**
+ * Holds methods shared by many RollupTest classes
+ */
+public class BaseRollupTest {
+
+    protected <T> Points<T> asPoints(Class<T> type, long initialTime, long timeDelta, T... values) {
+        Points<T> points = new Points<T>();
```

Comment:
They are the same exact copy. Previously, the tests in package com.rackspacecloud.blueflood.io.serializers use the BluefloodEnumRollupTest.asPoints() (the BluefloodEnumRollupTest is in com.rackspacecloud.blueflood.types). I didn't like that unnecessary dependency across packages like that. So I moved the BluefloodEnumRollupTest.asPoints() into a BaseRollupTest class. In the future, when we clean up the RollupTests, we can reuse this method.